File: MicroServices/Cinema.Services.SessionAPI/Models/Dtos/SeatSessionStatusDto.cs
using SharedLibrary.Enums;

namespace Cinema.Services.SessionAPI.Models.Dtos
{
    public class SeatSessionStatusDto
    {
        public int Id { get; set; }
        public int SessionId { get; set; }
        public int SeatId { get; set; }
        public string SeatNumber { get; set; }
        public ReservedStatusEnum ReservedStatus { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime UpdatedDate { get; set; }
    }
}

File: src/client/d-tix/src/models/session/SeatSessionStatusDto.ts
import {ReservedStatusEnum} from "@/models/enums/ReservedStatusEnum";

export default class SeatSessionStatusDto{
     sessionId: number;
     seatId: number;
     seatNumber: string;
     reservedStatus: ReservedStatusEnum;
     createdDate: Date;
     updatedDate?: Date;
     userId?: number;
}